gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/setBilling.ts

```diff
@@ -36,39 +35,79 @@ export default class SetBilling extends SolanaCommand {
     this.requireFlag('state', 'Provide a valid state address')
   }
 
-  execute = async () => {
+  makeRawTransaction = async (signer: PublicKey) => {
     const ocr2 = getContract(CONTRACT_LIST.OCR_2, '')
     const address = ocr2.programId.toString()
     const program = this.loadProgram(ocr2.idl, address)
-
     const state = new PublicKey(this.flags.state)
-    const input = this.makeInput(this.flags.input)
+
+    const input = this.makeInput(
+      ({
```

Comment:
Forgot to remove this. It should be: `const input = this.makeInput(this.flags.input)`